ts/features/fims/saga/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { SagaIterator } from "redux-saga";
+import { call, put, select, takeLatest } from "typed-redux-saga";
+import { ActionType } from "typesafe-actions";
+import { fimsTokenSelector } from "../../../store/reducers/authentication";
+import { fimsDomainSelector } from "../../../store/reducers/backendStatus";
+import {
+  mockHttpNativeCall,
+  mockSetNativeCookie
+} from "../../pushNotifications/sagas/mockFIMSCallbacks";
+import {
+  fimsGetConsentsListAction,
+  fimsGetRedirectUrlAndOpenBrowserAction
+} from "../store/actions";
+import { fimsCTAUrlSelector } from "../store/selectors";
+
+export function* watchFimsSaga(): SagaIterator {
+  yield* takeLatest(
+    fimsGetConsentsListAction.request,
+    handleFimsGetConsentsList
+  );
+  yield* takeLatest(
+    fimsGetRedirectUrlAndOpenBrowserAction.request,
+    handleFimsGetRedirectUrlAndOpenBrowser
+  );
+}
+
+function* handleFimsGetConsentsList() {
+  // action: ActionType<(typeof fimsGetConsentsListAction)["request"]>
+  // ) {
+  //   NavigationService.navigate(FIMS_ROUTES.MAIN, {
+  //     screen: FIMS_ROUTES.CONSENTS
+  //   }); // TODO:: maybe move navigation in saga from utils
+
+  const fimsToken = yield* select(fimsTokenSelector);
+  const oIDCProviderUrl = yield* select(fimsDomainSelector);
```

Comment:
I prefer using the acronym, since it gives more context and it alone acts as a little crumb of documentation for "hey we are using OpenID Connect"

what do you think?

Reply:
Settled for `oidcProviderUrl `